_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,22 +24,37 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
+This rule applies to each `content` attribute for which all of the following are true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the attribute is defined on a `meta` element with a `name` [attribute value][] of `viewport`; and
+- the [attribute value][] has either the `maximum-scale` or `user-scalable` key.
```

Comment:
Doesn't this make the following attribute inapplicable?

```html
<meta name="viewport" content="maximum-scale=2, user-scalable=5" />
```

It doesn't have either of the attributes. It has both. 
And I believe this should be applicable.